server/tags.json.example

```diff
@@ -0,0 +1,53 @@
+[
+    {
+        "id": "pushups",
+        "fields": [
+            {
+                "name": "sets",
+                "type": "int"
+            },
+            {
+                "name": "total",
+                "type": "int"
+            }
+        ]
+    },
+    {
+        "id": "enoughsleep",
+        "fields": [
+            {
+                "name": "value",
+                "type": [
+                    "yes",
+                    "no"
+                ]
+            }
+        ]
+    },
+    {
+        "id": "vacation",
+        "fields": [
+            {
+                "name": "location",
+                "type": "string"
+            },
+            {
+                "name": "enddate",
+                "type": "date"
+            }
+        ]
+    },
+    {
+        "id": "run",
+        "fields": [
+            {
+                "name": "km",
+                "type": "float"
+            },
+            {
+                "name": "time",
+                "type": "time"
```

Comment:
There was a fault in type validation logic for time fields. Now the correct time is represented as ISO 8601 date string with date set to 0001-01-01.